packages/discord.js/typings/index.d.ts

```diff
@@ -2967,10 +2968,18 @@ export class WebSocketShard extends EventEmitter {
   ): this;
 }
 
+export type WidgetStyleOptions =
+  | GuildWidgetStyle.Banner1
+  | GuildWidgetStyle.Banner2
+  | GuildWidgetStyle.Banner3
+  | GuildWidgetStyle.Banner4
+  | GuildWidgetStyle.Shield;
+
 export class Widget extends Base {
   private constructor(client: Client, data: RawWidgetData);
   private _patch(data: RawWidgetData): void;
   public fetch(): Promise<Widget>;
+  public fetchImage(style?: WidgetStyleOptions): Promise<Widget>;
```

Comment:
```suggestion
export class Widget extends Base {
  private constructor(client: Client, data: RawWidgetData);
  private _patch(data: RawWidgetData): void;
  public fetch(): Promise<Widget>;
  public fetchImage(style?: GuildWidgetStyle): Promise<Widget>;
```

The `WidgetStyleOptions` type alias is redundant since it contains all members of the `GuildWidgetStyle` enum.